apps/application-system/api/src/app/modules/application/application.module.ts

```diff
@@ -46,14 +47,18 @@ if (process.env.INIT_SCHEMA === 'true') {
   })
 }
 
+@Global()
```

Comment:
Would be best to refactor some of this "app" into "libs" which are easier to share without circular dependencies.

Reply:
Yes agree that would be preferable. Could we let this one slip by for now? I would like to create another PR with this refactor.

Reply:
Totally!